webapp/channels/src/packages/mattermost-redux/src/reducers/entities/cloud.test.ts

```diff
@@ -16,7 +16,7 @@ const minimalLimits = {
 
 describe('limits reducer', () => {
     test('returns empty limits by default', () => {
-        expect(limits(undefined, {type: 'some action', data: minimalLimits})).toEqual({
+        expect(limits(undefined, {type: undefined})).toEqual({
```

Comment:
Why these changes?

Reply:
I didn't split this part out quite right when I broke up these changes since it isn't technically needed until #29486.

My end goal with this set of changes is to make it so that we can define types for some actions and treat the rest like `AnyAction` as they do currently, but unfortunately to do that, I had to list every possible value of `type` which means that we can't dispatch arbitrary actions here to initialize the store. I had also tried to make this an empty string since that feels more correct, but it caused problems with type inference, so I went with `{type: undefined}` which some of these tests already used